flake.nix

```diff
@@ -269,7 +269,7 @@
               name = "treefmt";
               text = ''
                 unset PRJ_ROOT
-                ${lib.getExe (inputs.treefmt-nix.lib.mkWrapper pkgs treefmtConfig)}
+                ${lib.getExe (inputs.treefmt-nix.lib.mkWrapper pkgs treefmtConfig)} "$@"
```

Comment:
can treefmt handle an empty string as an argument?

Reply:
Bash handles `"$@"` specially, so it won't pass an empty string when there are no arguments.